### DISCUSSION TITLE
Add setting to change API port

### DISCUSSION THREAD

**Discussion by danielwerg** [massCodeIO/massCode#329] [Ideas]
https://github.com/massCodeIO/massCode/blob/c8b1c6a4be754bf3f1efade02599c03c65d1cb2e/src/main/config.ts#L1

**Comment by ondrovic** [comment]
This would be great to have, especially for windows users as that port sometimes gets blocked by something. I manually reconfigured the port and built locally and haven't had the issue since. 

Issue Related: https://github.com/massCodeIO/massCode/issues/481

**Comment by jagmagana** [comment]
This would be perfect, in my company the port 3033 is blocked, and it's imposible to use masscode here.

Thanks!

**Comment by ondrovic** [comment]
Still present in v4+ even though it seems the port have been changed to 4321, a simple preference option to change the port wouldn't be the easiest fix

<img width="2126" height="562" alt="image" src="https://github.com/user-attachments/assets/ca537609-fcf7-4a67-93ec-28313b74d5ad" />

I just put in this bug and have forked the repo hopefully can put in a PR to add this
https://github.com/massCodeIO/massCode/issues/630

PR: https://github.com/massCodeIO/massCode/pull/631

**Reply by ondrovic** [reply]
The above was merged, and you can either build from source or wait until the next release, this should resolve the issues with port blocking that everyone is having

**Comment by danielwerg** [comment]
I have migrated to [nap](https://github.com/maaslalani/nap) sometime after opening this discussion, but thank you for PR regardless :'D
